x-pack/filebeat/module/coredns/_meta/fields.yml

```diff
@@ -0,0 +1,60 @@
+- key: coredns
+  title: Coredns
+  description: >
+    Module for handling logs produced by coredns
+  fields:
+  - name: coredns
+    type: group
+    description: >
+      coredns fields after normalization
+    fields:
+    - name: id
+      type: keyword
+      description: >
+        id of the DNS transaction
+
+    - name: query.size
+      type: integer
+      description: >
+        size of the DNS query
+
+    - name: query.class
+      type: keyword
+      description: >
+        DNS query class
+
+    - name: query.name
+      type: keyword
+      description: >
+        DNS query name
+
+    - name: query.type
+      type: keyword
+      description: >
+        DNS query type
+
+    - name: logging_level
```

Comment:
Is this `log.level` from ECS?

@webmat FYI

Reply:
Seems to be a good fit there. will use that.